Distribution/Server/Features/Users.hs

```diff
@@ -468,6 +493,66 @@ userFeature  usersState adminsState
           errBadRequest "User deleted"
             [MText "Cannot disable account, it has already been deleted"]
 
+    serveUserManagement :: DynamicPath -> ServerPartE Response
+    serveUserManagement dpath =
+        do (UserName username) <- userNameInPath dpath
```

Comment:
in lots of other places we simplified this to:

    uid <- lookupUserName =<< userNameInPath dpath

Also, as a style thing we usually put the 'do' on the previous line, ie

    serveUserManagement dpath = do
        ... etc